tests/vmi_servers.go

```diff
@@ -18,8 +18,12 @@ const (
 	HTTPServer = server("\"HTTP/1.1 200 OK\\nContent-Length: 12\\n\\nHello World!\"\n")
 )
 
-func (s server) composeNetcatServerCommand(port int) string {
-	return fmt.Sprintf("screen -d -m sudo nc -klp %d -e echo -e %s", port, string(s))
+func (s server) composeNetcatServerCommand(port int, extraArgs ...string) string {
```

Comment:
There is a lie at `extraArgs` we are just using the first one let's just pass one string with the `expandNc` name or the like.

Reply:
Changed to `strings.Join(extraArgs, " ")`, so all the args will be used.